cocos/spine/assembler/simple.ts

```diff
@@ -261,31 +257,19 @@ function updateComponentRenderData (comp: Skeleton, batcher: Batcher2D) {
     _mustFlush = true;
     _premultipliedAlpha = comp.premultipliedAlpha;
     _multiplier = 1.0;
-    _handleVal = 0x00;
     _needColor = false;
     _vertexEffect = comp._effectDelegate && comp._effectDelegate._vertexEffect as any;
 
     if (nodeColor._val !== 0xffffffff ||  _premultipliedAlpha) {
         _needColor = true;
     }
 
-    if (_useTint) {
-        _handleVal |= FLAG_TWO_COLOR;
```

Comment:
Shouldn't remove this

Reply:
> Shouldn't remove this

no used _handleVal  with FLAG_TWO_COLOR. use _useTint flag directly.